conf/deployment/fusion_hci_pc/client_bm_upi_1az_rhcos_nvme_3m_3w.yaml

```diff
@@ -0,0 +1,13 @@
+---
+DEPLOYMENT:
+  allow_lower_instance_requirements: false
+  local_storage: true
+ENV_DATA:
+  platform: 'baremetal'
+  cluster_type: 'hci_client'
+  cluster_namespace: "openshift-storage"
```

Comment:
What about the namespace change? Don't we want to specify here the new client namespace?

Reply:
good catch. Copy paste mistake. Correcting it